src/models/StakingBalance.ts

```diff
@@ -0,0 +1,27 @@
+import { Address, BigInt } from '@graphprotocol/graph-ts'
+import { StakingBalance } from '../../generated/schema'
+
+export function getOrCreateStakingBalance(
+  account: Address,
+  contractAddress: Address,
+): StakingBalance {
+  let id = contractAddress.toHexString() + account.toHexString()
```

Comment:
This id is the same as the `getOrCreateStakingReward` fn. Does that matter? If so, change it.

Reply:
It should be what we want, because this is the id of the table this entity is in; the other is a different table :+1: